codecov.yml

```
@@ -5,4 +5,4 @@ coverage:
         target: 90%
     patch:
       default:
-        target: 80%
\ No newline at end of file
+        target: 60%
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)